pilot/pkg/proxy/envoy/v2/constants.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2018 Istio Authors
```

Comment:
I don't usually argue about file names - but when I do it's for files named 'constants'...

IMHO constants used in only one file should be in that file, for readability - opening a different file to check the constant value is a pain. 

I think shared constants and shared types/interfaces and initialization can go to the same file.

Reply:
Of course, deferring to  Zack on naming :-)